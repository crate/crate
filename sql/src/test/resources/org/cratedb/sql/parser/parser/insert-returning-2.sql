insert into t1 values (default, 'fred') returning *