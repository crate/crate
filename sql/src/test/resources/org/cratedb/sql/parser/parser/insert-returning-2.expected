org.cratedb.sql.parser.parser.InsertNode@31884174
statementType: INSERT
resultSet: 	
	org.cratedb.sql.parser.parser.RowResultSetNode@48cbdb20
	correlation Name: null
	null
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@3f9f3736

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@6de1dadb
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: true
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.DefaultNode@6e79839
			columnName: null
			defaultText: null
			type: null
		[1]:		
		org.cratedb.sql.parser.parser.ResultColumn@4ebac9b9
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.CharConstantNode@4911b910
			value: fred
			type: CHAR(4) NOT NULL
targetTableName: 	
	org.cratedb.sql.parser.parser.TableName@e3d
	t1
returningList: 	
	org.cratedb.sql.parser.parser.ResultColumnList@2a8ceeea

	[0]:	
	org.cratedb.sql.parser.parser.AllResultColumn@a995a79
	tableName: null
	exposedName: null
	name: null
	tableName: null
	isDefaultColumn: false
	type: null